GUIDELINES.md

```diff
@@ -52,7 +52,7 @@ The user interface must be clear, concise, and easy to use, leaving no room for
 - Check the search functionality to ensure other presets do not cause confusion.
 - Select an icon or start the process to create a new one.
 - Define which fields to show (`fields`) and suggest (`moreFields`), considering the order of fields.
-- Check the `(i)` documentation and add or update the Wikidata item if needed to provide a helpful short text.
+- Check the `(i)` documentation and add or update the OSM wiki data item if needed to provide a helpful short text.
```

Comment:
```suggestion
- Check the `(i)` documentation and add or update the OSM Wikidata item if needed to provide a helpful short text.
```

For consistency with https://github.com/openstreetmap/id-tagging-schema/pull/1257/files